apps/prairielearn/src/api/trpc/routers/course-files/batchDeleteQuestions.sql

```diff
@@ -0,0 +1,8 @@
+-- BLOCK select_questions_by_ids_and_course_id
+SELECT
+  *
+FROM
+  questions as q
+WHERE
+  q.id::text = ANY ($question_ids)
```

Comment:
This is a better way to write this:

```suggestion
  q.id = ANY ($question_ids::bigint[])
```

Reply:
Ahh, I was wondering if there were perf concerns over the cast. Switched in 253c7cd.